users.yml

```diff
@@ -0,0 +1,5 @@
+users:
```

Comment:
Maybe add `users.yml` to `.gitignore`, so we don't accidentally commit configuration files (they don't necessarily belong on git)

Reply:
👍🏻

Reply:
Same as `audit.log`